File: app/src/main/java/com/supercom/puretrack/model/business_logic_models/network/sync_requests/OffenderRequestType.java
package com.supercom.puretrack.model.business_logic_models.network.sync_requests;

public interface OffenderRequestType {
    int MESSAGE = 1;
    int SYNC = 2;
    int BIOMETRIC = 3;
    int SYNC_NOW = 4;
    int TERMINATE = 5;
    int SUSPEND = 6;
    int ACTIVATE = 7;
    int SW_UPGRADE = 8;
    int MANUAL_HANDLE = 10;
    int REMOTE_COMMAND = 14; // flight mode, reboot, etc.
    int SUSPEND_RESUME = 15;
    int PHOTO_ON_DEMAND_MESSAGE = 18;
}

File: app/src/main/java/com/supercom/puretrack/data/cycle/interfaces/OffenderRequestType.kt
package com.supercom.puretrack.data.cycle.interfaces

interface OffenderRequestType {
    companion object {
        const val MESSAGE = 1
        const val SYNC = 2
        const val BIOMETRIC = 3
        const val SYNC_NOW = 4
        const val TERMINATE = 5
        const val SUSPEND = 6
        const val ACTIVATE = 7
        const val SW_UPGRADE = 8
        const val MANUAL_HANDLE = 10
        const val REMOTE_COMMAND = 14
        const val SUSPEND_RESUME = 15
        const val PHOTO_ON_DEMAND_MESSAGE = 18
    }
}